tests/api/test_story.py

```diff
@@ -1173,10 +1180,296 @@ def test_get_stories(client, resource, uids, expected):
     for uid in uids:
         url = '/api/v1/story/{0}/{1}'.format(resource, uid)
         rv = client.get(url)
-        assert rv.status_code == 200, 'Failed getting the resource at: {0}'.format(url)
+        assert rv.status_code == 200
         assert json.loads(rv.data.decode('utf-8')) == expected
 
 
+@pytest.mark.parametrize('resource,uids,expected', [
+    ('freshmakerevent', ['1180'], {
```

Comment:
You only use parametrize for one test, so this could be converted to a traditional test for better readability.

Reply:
Nevermind, we can keep this as is. It'll make it easier to expand later.